### PR TITLE
fix(mux-viewer-count): Add A11Y. Default contents to allow tab focus.

### PR DESCRIPTION
**Pull request by cjpillsbury**
- Adds a11y
- Adds enabled/disabled (relates to a11y for tab focus behavior)
- Adds default contents for `viewer-count` (relates to a11y for SR + focus behavior of empty vs. non-empty nodes)

### CODE REVIEW

File: packages/mux-viewer-count/src/index.ts [hunk 1]
```diff
@@ -123,18 +132,39 @@ class MuxViewerCountElement extends globalThis.HTMLElement {
...
+  }
+
   connectedCallback() {
+    if (!this.hasAttribute('disabled')) {
+      this.enable();
+    }
+
+    /** @TODO Update to use `viewerCount` prop when available (CJP) */
+    this.setAttribute('aria-label', toAriaLabel({}));
+    this.setAttribute('role', 'presentation');
```
Comment (gkatsev) [inline packages/mux-viewer-count/src/index.ts:150]: this can probably be in the constructor, but probably doesn't matter that much.

**Review by gkatsev — APPROVED** [review]

File: packages/mux-viewer-count/src/index.ts (hunk 1 above)
Comment (cjpillsbury) [reply packages/mux-viewer-count/src/index.ts:150]: We probably don't want to do that. Per spec:

- The element's attributes and children must not be inspected, as in the non-[upgrade](https://html.spec.whatwg.org/multipage/custom-elements.html#upgrades) case none will be present, and relying on upgrades makes the element less usable.
- The element must not gain any attributes or children, as this violates the expectations of consumers who use the [createElement](https://dom.spec.whatwg.org/#dom-document-createelement) or [createElementNS](https://dom.spec.whatwg.org/#dom-document-createelementns) methods.

\- From: https://html.spec.whatwg.org/multipage/custom-elements.html#custom-element-conformance

Comment (gkatsev) [reply packages/mux-viewer-count/src/index.ts:150]: oh, right, I always forget that. Eitherway. :shipit:

**Comment by cjpillsbury** [comment]
@jaredsmith any reason not to merge this?

**Review by jaredsmith — APPROVED** [review]
Looks good to me.

**Pull request merged**